### PR TITLE
perf(ember): remove unsaved project from store without refetch

### PR DESCRIPTION
**Pull request by C0rydoras**

### CODE REVIEW

File: ember/app/routes/projects/index.js [hunk 1]
```diff
@@ -5,7 +5,10 @@ export default class ProjectsRoute extends Route {
   @service store;
 
   beforeModel() {
-    this.store.unloadAll('project');
+    const unsavedProject = this.store
+      .peekAll('project')
+      .find((project) => !project.status);
```
Comment (anehx) [inline ember/app/routes/projects/index.js:10]: I think there is an `isNew` property for exactly this use case

Comment (anehx) [reply ember/app/routes/projects/index.js]: Did you test it afterwards?

Comment (C0rydoras) [reply ember/app/routes/projects/index.js]: Implemented here https://github.com/adfinis/Outdated/pull/68/commits/54d10f40f103abdb7c67b02fc6957c6a9981060f

Comment (C0rydoras) [reply ember/app/routes/projects/index.js]: tested before and after

**Review by anehx — APPROVED** [review]

**Pull request merged**